src/main/scala/temple/generate/kube/ast/OrchestrationType.scala

```diff
@@ -9,10 +9,20 @@ object OrchestrationType {
     * Describes one microservice deployment in Kubernetes
     * @param name Service Name
     * @param image The name *including registry* of the docker image for this service
+    * @param dbImage The name of the docker image to use for the service's datastore
     * @param ports The ports the Pod should expose, including the names of the ports (i.e www -> 80)
     * @param replicas The number of replicas of the pod that should be exposed
-    * @param secretName The name of the Kubernetes secret used to fetch images from the registr
+    * @param secretName The name of the Kubernetes secret used to fetch images from the registry
+    * @param envVars A sequence of key -> value to set as environment variables in the container
     */
-  case class Service(name: String, image: String, ports: Seq[(String, Int)], replicas: Int, secretName: String)
+  case class Service(
+    name: String,
+    image: String,
+    dbImage: String,
+    ports: Seq[(String, Int)],
```

Comment:
Is there an advantage to using a `Seq` over a `Map`?

Reply:
We only ever need to iterate/map over the ports so a list of tuples is really all we need, however I'm happy to change to using Map[K,V] everywhere instead of Seq[(K, V)] If we think it's more readable

Reply:
That's cool, was more curiosity than anything else